### ISSUE TITLE
Selecting and copying issue

### ISSUE THREAD

**Issue by rugk** [mozilla/pdf.js#6658]
example: https://www.ietf.org/proceedings/82/slides/rtcweb-13.pdf (take side 2)
1. When copying the normal text everything works (expect that some characters like { and � seem to be inserted in some places).
   If I copy the site two it looks like this:

```
Trust Model

Browser acts as the
Trusted Computing Base
(TCB)
{
Only piece of the system user can really trust
{
Job is to enforce user's desired security policies

Authenticated entities
{
Identity is checked by the browser (sometimes transitively)

Unauthenticated entities
{
Random other network elements who send and receive tra
```
1. Selecting the whole site is also not really possible:
   ![markmore](https://cloud.githubusercontent.com/assets/11966684/11256777/341b1fc8-8e4e-11e5-806f-f062e764f911.gif)

Tested with Firefox 42

**Comment by Snuffleupagus** [comment]
> expect that some characters like { and � seem to be inserted in some places

This part is caused by the PDF itself, and *not* PDF.js, since copy-paste from Adobe Reader produces the *exact* same characters. (The only difference being that the PDF.js copy-paste result contains a number of unwanted extra line-breaks.)

**Comment by Snuffleupagus** [comment]
> When copying the normal text everything works (expect that some characters like { and � seem to be inserted in some places).

This was fixed by PR #7550.

> Selecting the whole site is also not really possible:

This was fixed by PR #7879.

Closing, since all the points reported above should now have been fixed.
